include/inexor/vulkan-renderer/renderer.hpp

```diff
@@ -161,15 +160,8 @@ class VulkanRenderer {
     // TODO(Hanni): Remove this with RAII refactoring of descriptors!
     VkDescriptorImageInfo descriptor_image_info = {};
 
-    /// @brief Creates a Vulkan instance.
-    /// @param application_name The name of the application
-    /// @param engine_name The name of the engine.
-    /// @param application_version The version of the application encoded as an unsigned 32 bit integer.
-    /// @param engine_version The version of the engine encoded as an unsigned 32 bit integer.
-    /// @param enable_validation_layers True if validation is enabled.
-    VkResult create_vulkan_instance(const std::string &application_name, const std::string &engine_name, const std::uint32_t application_version,
-                                    const std::uint32_t engine_version, bool enable_validation_instance_layers = true,
-                                    bool enable_renderdoc_instance_layer = false);
+    // RAII wrapper for VkInstance.
+    std::unique_ptr<wrapper::Instance> vkinstance;
```

Comment:
We need to control the point of construction, because some constructor parameters of `vkinstance` are not known yet when we would have to construct it in `VulkanRenderer`'s constructor.

Reply:
done.